README.md

```
@@ -36,15 +36,16 @@
 	<br />
 	<a href="https://scaleflex.github.io/filerobot-image-editor/">Editor Preview</a>
   •
-	<a href="https://www.filerobot.com/en/home">Learn more about Filerobot</a>
+	<a href="https://www.filerobot.com/en/home">Learn more about Scaleflex
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)